styles/main.scss

```diff
@@ -0,0 +1,83 @@
+$base: #ffffff;
+$navbar: #cccccc;
+body {
```

Comment:
Consider if anything needs to be nested under the `body` tag. Putting things under the `body` tag applies styles to everything. Putting any tags under it is kinda pointless.